mega-zep-frontend/src/main/angular/frontend/angular.json

```diff
@@ -30,7 +30,8 @@
             "styles": [
               "./node_modules/@angular/material/prebuilt-themes/deeppurple-amber.css",
               "src/styles.scss",
-              "node_modules/bootstrap/dist/css/bootstrap.css"
+              "node_modules/bootstrap/dist/css/bootstrap.css",
```

Comment:
we should check if we can use the scss variant of bootstrap, so that everything is scss